### DISCUSSION TITLE
Improve SSL request

### DISCUSSION THREAD

**Discussion by omerlh** [projectdiscovery/nuclei#1462] [Ideas]
I couldn't find the documentation for SSL request on the website, but looking at the code I see it is limited in the information passed.
Ideally I would like to do something similar to SSL Labs with Nuclei (testing cipher suites for weak ciphers, for a start). I think this is not possible today as this data is not exposed - but I am wonder if it is possible to add it?

**Comment by ehsandeep** [comment]
You are right @omerlh, support for SSL protocol was added recently and we still have to update documentation for new additions, currently, SSL protocol exposes limited information and intends to extend and expose more information to cover more cases.

Some discussion around it - https://github.com/projectdiscovery/nuclei/issues/1231

**Reply by omerlh** [reply]
Ahh sure! I saw the issue but I think it is more about supporting more ciphers and less about detecting weak ciphers - unless I am mistaken

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
